decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -171,6 +171,10 @@ def bundle_install
         run "bundle install"
       end
 
+      def precompile_assets
```

Comment:
We need to compile assets before seeding, otherwise the seeds will not work, as it fails with `Shakapacker can't find decidim_email.css` error.

![image](https://github.com/decidim/decidim/assets/105683/0e109b0e-1e77-443f-a9cc-a753fe5c9219)

Reply:
Ah OK, I understand this didn't happen before as we weren't firing the ActiveJob jobs. Thanks for the explanation!